tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -109,6 +112,13 @@ context('Multiple users. Assign task, job. Deactivating users.', () => {
         it('First user login, create a task and logout', () => {
             cy.login();
             cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
+            cy.goToTaskList();
+            cy.openTaskJob(taskName);
+            // Getting the task and job id
+            cy.url().then((url) => {
+                jobID = Number(url.split('/').slice(-1)[0]);
+                taskID = Number(url.split('/').slice(-3)[0]);
+            });
             cy.logout();
```

Comment:
Should we really logout before the previous it()? We login again there

Reply:
Maybe unite these cases?

Reply:
Sure. Thanks.